libs/books/data-access/src/lib/+state/reading-list.effects.spec.ts

```diff
@@ -42,4 +51,23 @@ describe('ToReadEffects', () => {
       httpMock.expectOne('/api/reading-list').flush([]);
     });
   });
+
+  describe('markBookAsFinished$', () => {
+    it('should mark the book as finished when user clicks on button ', done => {
+      actions = new ReplaySubject();
+      const Item = createReadingListItem('A');
+      const finishedDate = new Date().toISOString();
+      actions.next(ReadingListActions.markBookAsFinished({ item: Item, finishedDate: finishedDate }));
+
+      effects.markBookAsFinished$.pipe(takeUntil(unsubscribe$))
```

Comment:
Why we need `takeUntil(unsubscribe$)` in test case.

Reply:
used takeUntil so that we can cancel the subscribe of observable. Seems it is not needed. Removed takeUntil